R/racing_helpers.R

```diff
@@ -387,7 +387,7 @@ harmonize_configs <- function(x, key) {
   x
 }
 
-restore_tune <- function(x, y) {
+restore_tune <- function(x, y, eval_time_target = NULL) {
```

Comment:
`x` and `y` are both `tune_results` objects, right? do they now both carry `eval_time_target` as an attribute that could be used here instead of providing it as an argument?

Reply:
`x` has class `tune_results`. `y` has the same structure but different attributes. The extra classes have been dropped due to row filtering of the resampling object. The function restores missing attributes to `y`. 

However... `x` is from `tune_grid()`, which has no notion of a target evaluation time. https://github.com/tidymodels/tune/pull/782 defaults `eval_time_target` to NULL for grid tuning, resampling, and last fit objects. 

That's why `eval_time_target` is an argument to this function. It should be non-null for the resulting racing object but the value inherited from `x` is NULL unless we set it.

Reply:
Ah, thanks! 👍

Reply:
notes added to sources